### ISSUE TITLE
[Client Step 1.3] change package.json

### ISSUE THREAD

**Issue by czd** [Urigo/WhatsApp-Clone-Tutorial#12]
Running `yarn format` on Windows produces the error "No matching files." due to the single quotes. It works when removing the quotes in the command.

```json
{
  "name": "whatsapp-clone-client",
  "version": "0.1.0",
  "private": true,
  "repository": {
    "type": "git",
    "url": "https://github.com/Urigo/WhatsApp-Clone-Client-React.git"
  },
  "dependencies": {
    "@types/jest": "24.0.15",
    "@types/node": "12.0.10",
    "@types/react": "16.8.22",
    "@types/react-dom": "16.8.4",
    "prettier": "1.18.2",
    "react": "16.8.6",
    "react-dom": "16.8.6",
    "react-scripts": "3.0.1",
    "typescript": "3.5.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "TZ=\"Asia/Jerusalem\" react-scripts test",
    "eject": "react-scripts eject",
    "format": "prettier **/*.{ts,tsx,css,graphql} --write"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

**Comment by Urigo** [comment]
ohh true, we've fixed it on the server already - #3 

Probably a better fix would be to escape it as in there no?
https://github.com/Urigo/WhatsApp-Clone-Server/blob/master/package.json#L17

**Comment by Urigo** [comment]
I've fixed it and deployed a new version and added you to the contributors on the bottom of the README

Thank you!
